### ISSUE TITLE
DATABASE : ARANGODB 3.3.x - possible integration ?

### ISSUE THREAD

**Issue by ephysthos** [metabase/metabase#7867]
Dear support, 

As you know, ArangoDB start to be very interesting projet for lot of compagnies and individuals. Very powerfull, it misses actually your wonderfull product for bypassing microsoft powerbi devil 🍡 

A java driver exists : https://www.arangodb.com/arangodb-drivers/

I'm not a java developper, but do you think it will be possible to integrate this driver as mongodb is ? 

Thanks a lot for your contribution, 

BR, 
Guillaume.

**Comment by ephysthos** [comment]
Dear support, 
any news for my proposal ? some people seems to be very interested about it :)

Thanks

**Comment by mysticaltech** [comment]
It would be great to have support for Arango!

**Comment by starxyn** [comment]
Hi,

Has Metabase implemented arangodb driver for metabase now?

**Comment by dragonsahead** [comment]
@starxyn no, there's no demand from any current customer for this database
